tests/common/util.rs

```diff
@@ -140,6 +214,13 @@ impl CmdResult {
         Box::new(self)
     }
 
+    /// asserts that the command resulted in stderr stream output,
+    /// whose bytes equal those of the passed in slice
+    pub fn stderr_is_bytes<T: AsRef<[u8]>>(&self, msg: T) -> Box<&CmdResult> {
+        assert_eq!(self.stderr.as_bytes(), msg.as_ref());
+        Box::new(self)
```

Comment:
Can you try to remove the `Box`es?  I think they must be a remnant of a change from long ago.

Reply:
sure thing